src/main/java/org/interledger/setup/spsp/model/Invoice.java

```diff
@@ -0,0 +1,15 @@
+package org.interledger.setup.spsp.model;

```

Comment:
shouldnt this just be org.interledger.setup.model? Are we expecting other setup models apart from SPSP?
for brevity, same comment applies to the other classes in this package (InvoiceStatus, Payee etc).